EffectivePotential/src/one_loop_potential.cpp

```diff
@@ -162,10 +162,24 @@ double OneLoopPotential::V1(std::vector<double> scalar_masses_sq,
   return correction / (64. * M_PI * M_PI);
 }
 
-double OneLoopPotential::V1(Eigen::VectorXd phi) const {
-  return V1(get_scalar_masses_sq(phi, xi),
-            get_fermion_masses_sq(phi),
-            get_vector_masses_sq(phi));
+double OneLoopPotential::V1(Eigen::VectorXd phi, double T) const {
+  if (T != 0 && daisy_method != DaisyMethod::Parwani) {
+    throw std::runtime_error("V1 dependent on T only with Parwani daisy method");
+  }
+
+  switch (daisy_method) {
+    case DaisyMethod::None:
+    case DaisyMethod::ArnoldEspinosa:
+      return V1(get_scalar_masses_sq(phi, xi),
```

Comment:
I am not used to a switch where one case has no statement, but I assume this syntax just gibes first two cases the statement. However since we only have three cases, wouldn't it be clearer to just use an if( daisy_method == DaisyMethod::Parwani){..} else {..} statement here?  

i suppose this can be more easily extended to other methods though?

Reply:
yes, this is how to implement logical OR in switch statements, as switch statements cascade down until a break or return. switch versus if/else seems to be moot, https://stackoverflow.com/questions/97987/advantage-of-switch-over-if-else-statement

Reply:
I don't care that much and its really the only time I am arguing this way round, as opposed to telling people to change spaghetti-like if-else chains to a switch.  However i would say if we were sure that we won't add a 4th option in future just use if-else, but if there is any possibility we'll extend it, just leave it as though maybe add a comment explaining that none amd ArnoldEspinosa are the same here.  Actually I guess in this case we also use a switch statement with only one case (Parwani) and a default for the other two. Somehow that sounds neater to me, but it could just be me.